tests/quality/config.yaml

```diff
@@ -25,7 +25,7 @@ yardstick:
       # Note:
       #  - ALWAYS leave the "import-db" annotation as-is
       #  - this version should ALWAYS match that of the other "grype" tool below
-      version: latest+import-db=build/grype-db.tar.gz
+      version: feat-azure-linux-3-support+import-db=build/grype-db.tar.gz
```

Comment:
before merging -- will need to change this value (+ below values)

Reply:
yep - it actually looks like static-analysis checks for this now, which is awesome.